### PR TITLE
feat(WalletConnect): change to public WalletConnect relay

### PR DESCRIPTION
**Pull request by Harasz**

### CODE REVIEW

**Review by JGiter — COMMENTED** [review]
Have you managed to connect with this bridge?

**Comment by Harasz** [comment]
I have with Volta RPC, but there is a problem with signing message bc method `eth_sign` is not supported.
On our dev RPC I haven't, dev RPC is not supporting `skipCache` param in request.

**Comment by JGiter** [comment]
I can't get to signing, because after QR code scanning there are endless requests to Volta

File: src/iam/iam-base.ts [hunk 1]
```diff
@@ -242,6 +245,8 @@ export class IAMBase {
             this._provider = new providers.Web3Provider(wcProvider);
             this._signer = this._provider.getSigner();
             this._providerType = walletProvider;
+            const walletConnector = this._walletConnectService.getConnector();
+            this._walletConnectSignMessage = walletConnector.signMessage.bind(walletConnector);
```
Comment (JGiter) [inline src/iam/iam-base.ts:249]: There a lot of places in code where we need signing. Wouldn't be better to implement Signer on top of wallet connect?

**Review by jrhender — CHANGES_REQUESTED** [review]
The fix looks good, nice work. It's annoying we needed to use `patch-package` but at least the change isn't very big.

Could you somehow point to the GitHub issue that you created in the code? Maybe squash your commit and add the link to the commit messages? My goal is just to have a way for future devs to know why this was put in and when they can try to remove it.

**Pull request merged**